pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic, ReferenceFrame
+from sympy.physics.mechanics import Particle, RigidBody,\
+    dynamicsymbols, outer
+
+# local
+from ..bodies import Body
+
+
+def test_points(point1, point2):
+    assert point1.name == point2.name
+    assert point1._pos_dict == point2._pos_dict
+    assert point1._vel_dict == point2._vel_dict
+    assert point1._acc_dict == point2._acc_dict
+
+
+def test_referenceframes(frame1, frame2):
+    assert frame1.name == frame2.name
+    assert frame1.str_vecs == frame2.str_vecs
+    assert frame1.indices == frame2.indices
+    assert frame1.varlist == frame2.varlist
+    assert frame1._dlist == frame2._dlist
+    assert frame1._ang_vel_dict == frame2._ang_vel_dict
+    assert frame1._ang_acc_dict == frame2._ang_acc_dict
+    assert frame1._dcm_dict == frame2._dcm_dict
+
+class TestBody():
+    def test_default(self):
+        body = Body('body')
+        assert body.name == 'body'
+        assert body.parent is None
+        assert body.child is None
+        assert body.force_list == []
+        assert body._coordinates == []
+        assert body._speeds == []
+        test_points(body._masscenter, Point('body_masscenter'))
+        assert body._mass == Symbol('body_mass')
+        test_referenceframes(body._frame, ReferenceFrame('body_frame'))
+        assert body._inertia == (outer(body._frame.x, body._frame.x),
+                                 body._masscenter)
+        assert isinstance(body.body, RigidBody)
+
+        rigidbody = body.body
+        test_points(rigidbody.masscenter, body._masscenter)
+        assert rigidbody.mass == body._mass
+        test_referenceframes(rigidbody.frame, body._frame)
+        assert rigidbody.inertia == body._inertia
+
+    def test_custom_rigid_body(self):
+        masscenter = Point('rigidbody_masscenter')
+        mass = Symbol('rigidbody_mass')
+        frame = ReferenceFrame('rigidbody_frame')
+        inertia = outer(frame.x, frame.x)
+        inertia_tuple = (inertia, masscenter)
+        self.rigidbody_body = Body('rigidbody_body', masscenter, mass, frame,
+                                   inertia_tuple)
```

Comment:
If a body's inertia is always defined with respect to it's mass center and reference frame, we shouldn't allow the user to pass in a generic inertia dyadic and point. You should simple accept the same arguments as the `inertia` function, minus the reference frame.